assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -29,21 +29,23 @@ import Data from 'googlesitekit-data';
 import LegacySearchConsoleDashboardWidgetKeywordTable from '../dashboard/LegacySearchConsoleDashboardWidgetKeywordTable';
 import DashboardModuleHeader from '../../../../components/dashboard/DashboardModuleHeader';
 import Layout from '../../../../components/layout/Layout';
-import { STORE_NAME } from '../../datastore/constants';
+import { MODULES_SEARCH_CONSOLE, DATE_RANGE_OFFSET } from '../../datastore/constants';
```

Comment:
I realize @tofumatt raised this in his review https://github.com/google/site-kit-wp/pull/2634#discussion_r561120025, but our convention has always been to leave the `STORE_NAME` unaliased within the scope of the module's datastore and components. If this is confusing, we can revisit it in a dedicated issue to iterate on this, but we need to keep this for consistency.

TL;DR – modules or core components importing _their own_ `STORE_NAME` should do so using the generic `STORE_NAME` constant. We only used the named variant for referencing stores external to the current context.

This will reduce the size of the diff here as well.